_pkgdown.yml

```diff
@@ -100,3 +102,8 @@ reference:
   - reconcile_yaml
   - check_yaml_in_sync
   - suppressSpecificWarning
+  - test_threads
+  - check_run_times
+  - delete_models
+  - check_nonmem_finished
```

Comment:
Did you decide you wanted to include `delete_models()`? Im fine with either, just wanted to make sure it was intentional

Reply:
Yeah, I flipped!  In the end, I think, given we export it, we should have it in the index in case someone goes looking for it.  I dunno, but the biggest decision is to export it, and that's been made.  We can always remove it from the pkgdown index later if we want.